tests/core/metric/string_processor/test_regex.py

```diff
@@ -0,0 +1,16 @@
+import pytest
+
+from flexeval.core.metric.string_processor import RegexExtractor
+
+
+@pytest.mark.parametrize(
+    ("before", "after"),
+    [
+        ("<think>reasoning</think>answer", "answer"),
+        ("answer", "answer"),
+        ("", ""),
```

Comment:
It seems that Deepseek-like models generate <think> tags as follows, with newlines included at the beginning and the end.
Can you consider adding such test cases here?

```
<think>
Okay, so let's…
...
</think>
```

Reply:
The above example was incorrect. In fact, there is two newlines after `</think>` token.
```
<think>
Okay, so let's…
...
</think>

My answer is A.
```